### PR TITLE
feat(Notification): Group related changes

### PR DESCRIPTION
**Pull request by porames**
thanks

### CODE REVIEW

File: src/models/posts.js [hunk 1]
```diff
@@ -0,0 +1,30 @@
+import _ from 'lodash'
+import mongoose from 'mongoose'
+
+const PostSchema = new mongoose.Schema({
+    date: {
```
Comment (rayriffy) [inline src/models/posts.js]: Please formatting all files correctly by according to ESLint rules.

File: src/routes/v1/announce/list.js [hunk 2]
```diff
@@ -6,76 +6,76 @@ import Announce from '../../../models/announce'
 const router = express.Router()
 
 router.get('/:page', async (req, res) => {
-  let payload = []
+	let payload = []
```
Comment (rayriffy) [inline src/routes/v1/announce/list.js]: Why all spacings are changed?

File: src/routes/v1/group/list.js [hunk 3]
```diff
@@ -3,25 +3,23 @@ import Group from '../../../models/group'
...
-
-    const fetchedGroup = groupList.map((group, i) => {
-      const ownerId = group.owner
-      const owner = User.getNameById(ownerId)
-      const ownerName = owner.profile.fullname
-
-      return {...group, owner: ownerName}
-    })
+    for (var index = 0; index < groupList.length; index++) {
+      var ownerId = groupList[index].owner
```
Comment (rayriffy) [inline src/routes/v1/group/list.js:11]: Security rules for Generic Object Injection Sink `eslint(security/detect-object-injection)` failed

File: src/services/notify.js [hunk 4]
```diff
@@ -4,37 +4,55 @@ import Expo from 'expo-server-sdk'
...
+    }
+  }
+  else {
+    subscribers = await Subscriber.find({ group: { $eq: group._id } }) //get subscribers
+  }
+  for (var i = 0; i < subscribers.length; i++) {
+    await Subscriber.findOneAndUpdate({ $and: [{ 'user.id': subscribers[i].user.id }, { group: { $eq: group._id } }] }, { $set: { newUpdate: true } })
+    if (Expo.isExpoPushToken(subscribers[i].user.token)) {
+      messages.push({
+        to: subscribers[i].user.token,
```
Comment (rayriffy) [inline src/services/notify.js:41]: Security rules for Generic Object Injection Sink `eslint(security/detect-object-injection)` failed

**Review by rayriffy — CHANGES_REQUESTED** [review]
API are migrating to **Node 12**, please rebase your repository before committing any changes.

**Comment by porames** [comment]
linting fixed (I guess)

**Comment by rayriffy** [comment]
Now, there're security warnings left from Codacy

**Comment by rayriffy** [comment]
There's a conflict in `package.json` and `yarn.lock` and a lot of new issue being alerted from Codacy. I will close this PR.

Please open a new PR separately when your forked repo is up-to-date.

**Pull request closed without merging**